OloEngine/src/OloEngine/Scripting/Lua/LuaScriptEngine.cpp

```
@@ -83,34 +79,131 @@ namespace OloEngine
+                }
+            }
+        }
     }
 
-    void LuaScriptEngine::OnUpdate(const Entity*, f32)
+    void LuaScriptEngine::OnDestroyEntity(Entity entity)
     {
-        // auto const& lua = *s_LuaData.LuaState;
-        // LUA_CALL(entity->GetName(), "OnUpdate", ts);
+        auto entityID = static_cast<u64>(entity.GetUUID());
+        if (auto const it = s_LuaData.EntityScriptInstances.find(entityID); it != s_LuaData.EntityScriptInstances.end())
```

Clang (14.0.6)

[warning] variable name 'it' is too short, expected at least 3 characters

(readability-identifier-length)